client/tsconfig.json

```diff
@@ -15,6 +15,8 @@
     "allowJs": true,
     "resolveJsonModule": true,
     "baseUrl": "./",
+    // Todo VS: is this really needed?
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
We actually don't want this - more [here](https://www.typescriptlang.org/tsconfig#suppressImplicitAnyIndexErrors) but this can indicate places where we need to be more explicit about typing / can cast